app/knight.py

```diff
@@ -0,0 +1,36 @@
+class Knight:
+
+    def __init__(self, knight: dict) -> None:
+        self.name = knight["name"]
+        self.power = knight["power"]
+        self.hp = knight["hp"]
+        self.armour = knight["armour"]
+        self.weapon = knight["weapon"]
+        self.potion = knight["potion"]
```

Comment:
it would be better to use external classes for armour, weapon, potion for better understanding what is it

Reply:
also would be nice to create a module for knight and his ammunition, so all related files will be on the same package